pasts.lv,"HTTPSConnectionPool(host='pasts.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='pasts.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
patmar.ee,"HTTPSConnectionPool(host='patmar.ee', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='patmar.ee', port=443): Read timed out. (read timeout=2)"
patro.cz,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
pau.edu.tr,"HTTPSConnectionPool(host='pau.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'pau.edu.tr' doesn't match either of '*.internal.pau.edu.tr', '*.pau.edu.tr', '*.pusula.pau.edu.tr'"")))","HTTPSConnectionPool(host='www.pau.edu.tr', port=443): Read timed out. (read timeout=2)"
pbh.gov.br,"HTTPSConnectionPool(host='pbh.gov.br', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa3f8c7d880>, 'Connection to pbh.gov.br timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pbh.gov.br', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa3f9c02760>, 'Connection to pbh.gov.br timed out. (connect timeout=2)'))"
pcfarm.ro,"HTTPSConnectionPool(host='pcfarm.ro', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))","HTTPSConnectionPool(host='pcfarm.ro', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))"
pclab.pl,"HTTPSConnectionPool(host='pclab.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='pclab.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
pdn.ac.lk,"HTTPSConnectionPool(host='pdn.ac.lk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3f787d3a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pdn.ac.lk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3f9eaeeb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
peatus.ee,"HTTPSConnectionPool(host='web.peatus.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='web.peatus.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
pedagogica.edu.co,"HTTPSConnectionPool(host='pedagogica.edu.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3faaa90d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pedagogica.edu.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3f7b764c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
pediatriepropraxi.cz,"HTTPSConnectionPool(host='pediatriepropraxi.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='pediatriepropraxi.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
penang.gov.my,"HTTPSConnectionPool(host='penang.gov.my', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3f9a75820>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='penang.gov.my', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3f7560550>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
penzin.rs,"HTTPSConnectionPool(host='www.penzin.rs', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.penzin.rs', port=443): Read timed out. (read timeout=2)"
penziony.cz,"HTTPSConnectionPool(host='penziony.cz', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='penziony.cz', port=80): Read timed out. (read timeout=2)"
peoples.ru,"HTTPSConnectionPool(host='peoples.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.peoples.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
peoplesbank.lk,"HTTPSConnectionPool(host='peoplesbank.lk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3fa32a070>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='peoplesbank.lk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3f99b21f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
perach.org.il,"HTTPSConnectionPool(host='perach.org.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='perach.org.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
perak.gov.my,"HTTPSConnectionPool(host='perak.gov.my', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3f99243d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='perak.gov.my', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3fa497070>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
peritiuniti.it,"HTTPSConnectionPool(host='peritiuniti.it', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='peritiuniti.it', port=80): Read timed out. (read timeout=2)"
persee.fr,"HTTPSConnectionPool(host='www.persee.fr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.persee.fr', port=443): Read timed out. (read timeout=2)"
petah-tikva.muni.il,"HTTPSConnectionPool(host='petah-tikva.muni.il', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa3f5fbd9d0>, 'Connection to petah-tikva.muni.il timed out. (connect timeout=2)'))","HTTPConnectionPool(host='petah-tikva.muni.il', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa3f94d3d00>, 'Connection to petah-tikva.muni.il timed out. (connect timeout=2)'))"
petitlyrics.com,"HTTPSConnectionPool(host='petitlyrics.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='petitlyrics.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
pgdlisboa.pt,"HTTPSConnectionPool(host='pgdlisboa.pt', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa3f49a69d0>, 'Connection to pgdlisboa.pt timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pgdlisboa.pt', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa3f4974610>, 'Connection to pgdlisboa.pt timed out. (connect timeout=2)'))"
pgr.gob.ni,"HTTPSConnectionPool(host='pgr.gob.ni', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3f7856ee0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='pgr.gob.ni', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3f44f5880>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
phacdochuabenh.com,"HTTPSConnectionPool(host='phacdochuabenh.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='phacdochuabenh.com', port=443): Read timed out. (read timeout=2)"
pharmaline.co.il,"HTTPSConnectionPool(host='pharmaline.co.il', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa3fa665fa0>, 'Connection to pharmaline.co.il timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pharmaline.co.il', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa3f48a44c0>, 'Connection to pharmaline.co.il timed out. (connect timeout=2)'))"
pharmilion.gr,"HTTPSConnectionPool(host='pharmilion.gr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='pharmilion.gr', port=443): Read timed out. (read timeout=2)"
phimmoizz.net,"HTTPSConnectionPool(host='phimmoizz.net', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='phimmoizz.net', port=80): Read timed out. (read timeout=2)"
phoenixcertifiedhomeinspections.com,"HTTPConnectionPool(host='www.phoenixcertifiedhomeinspections.com', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.phoenixcertifiedhomeinspections.com', port=80): Read timed out. (read timeout=2)"
phunuonline.com.vn,"HTTPSConnectionPool(host='phunuonline.com.vn', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa3f2d5e670>, 'Connection to phunuonline.com.vn timed out. (connect timeout=2)'))","HTTPConnectionPool(host='phunuonline.com.vn', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa3f2f86310>, 'Connection to phunuonline.com.vn timed out. (connect timeout=2)'))"
phymmoi.net,"HTTPSConnectionPool(host='phymmoi.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='phymmoi.net', port=443): Read timed out. (read timeout=2)"
pichinchacomunicaciones.com.ec,"HTTPSConnectionPool(host='pichinchacomunicaciones.com.ec', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa3f229a8e0>, 'Connection to pichinchacomunicaciones.com.ec timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pichinchacomunicaciones.com.ec', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa3f254c280>, 'Connection to pichinchacomunicaciones.com.ec timed out. (connect timeout=2)'))"
pikabu.ru,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
pikiwiki.org.il,"HTTPSConnectionPool(host='www.pikiwiki.org.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pikiwiki.org.il', port=443): Read timed out. (read timeout=2)"
pikk.ee,"HTTPSConnectionPool(host='www.pikk.ee', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pikk.ee', port=443): Read timed out. (read timeout=2)"
piluli.ru,"HTTPSConnectionPool(host='piluli.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='piluli.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
pirkka.fi,"HTTPSConnectionPool(host='pirkka.fi', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.pirkka.fi', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
piroshairandbeauty.ro,"HTTPConnectionPool(host='www.piroshairandbeauty.ro', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.piroshairandbeauty.ro', port=80): Read timed out. (read timeout=2)"
pitat.com,"HTTPSConnectionPool(host='pitat.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3f5ce7490>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='pitat.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3f929e790>: Failed to establish a new connection: [Errno 111] Connection refused'))"
pivnidenicek.cz,"HTTPSConnectionPool(host='pivnidenicek.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'pivnidenicek.cz' doesn't match 'www.pivnidenicek.cz'"")))","HTTPSConnectionPool(host='pivnidenicek.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'pivnidenicek.cz' doesn't match 'www.pivnidenicek.cz'"")))"
pizzasnella.it,"HTTPSConnectionPool(host='pizzasnella.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3fb8a7ac0>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='www.pizzasnella.it', port=80): Read timed out. (read timeout=2)"
planetazdorovo.ru,"HTTPSConnectionPool(host='planetazdorovo.ru', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='planetazdorovo.ru', port=443): Read timed out. (read timeout=2)"
pleasantgroveumc.net,"HTTPSConnectionPool(host='pleasantgroveumc.net', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3f1b8b460>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pleasantgroveumc.net', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3f24f9220>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
plovdiv.bg,"HTTPSConnectionPool(host='plovdiv.bg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='plovdiv.bg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
plzen.eu,"HTTPSConnectionPool(host='www.plzen.eu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.plzen.eu', port=443): Read timed out. (read timeout=2)"
pmda.go.jp,"HTTPSConnectionPool(host='pmda.go.jp', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa3f24a6fd0>, 'Connection to pmda.go.jp timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pmda.go.jp', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa3f24f97c0>, 'Connection to pmda.go.jp timed out. (connect timeout=2)'))"
pmnewsnigeria.com,"HTTPSConnectionPool(host='www.pmnewsnigeria.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pmnewsnigeria.com', port=443): Read timed out. (read timeout=2)"
pnc.gob.sv,"HTTPSConnectionPool(host='pnc.gob.sv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3fb136eb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pnc.gob.sv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3f5290730>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
pnp.co.za,"HTTPSConnectionPool(host='pnp.co.za', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3f34a85e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pnp.co.za', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3fa7df820>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
poder-judicial.go.cr,"HTTPSConnectionPool(host='poder-judicial.go.cr', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa3f229a460>, 'Connection to poder-judicial.go.cr timed out. (connect timeout=2)'))","HTTPConnectionPool(host='poder-judicial.go.cr', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa3f916b730>, 'Connection to poder-judicial.go.cr timed out. (connect timeout=2)'))"
poderjudicial.gob.ni,"HTTPSConnectionPool(host='poderjudicial.gob.ni', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa3f25bd580>, 'Connection to poderjudicial.gob.ni timed out. (connect timeout=2)'))","HTTPConnectionPool(host='poderjudicial.gob.ni', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa3fbd00a90>, 'Connection to poderjudicial.gob.ni timed out. (connect timeout=2)'))"
poduckun.net,"HTTPSConnectionPool(host='poduckun.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='poduckun.net', port=443): Read timed out. (read timeout=2)"
poipeplekkers.be,"HTTPSConnectionPool(host='poipeplekkers.be', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3fba2d790>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='poipeplekkers.be', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3f9060550>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
poisci.me,"HTTPSConnectionPool(host='poisci.me', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='poisci.me', port=80): Read timed out. (read timeout=2)"
polazak.rs,Exceeded 30 redirects.,Exceeded 30 redirects.
policia.gov.co,"HTTPSConnectionPool(host='policia.gov.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3f0316910>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='policia.gov.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3f8e17910>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
politikaplus.com,"HTTPSConnectionPool(host='politikaplus.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='politikaplus.com', port=80): Read timed out. (read timeout=2)"
polito.it,"HTTPSConnectionPool(host='polito.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3f3009d30>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='polito.it', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3f9e7fb80>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
polsl.pl,"HTTPSConnectionPool(host='polsl.pl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3f5d671f0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='polsl.pl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3f9e19ee0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
pom.go.id,"HTTPSConnectionPool(host='pom.go.id', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='pom.go.id', port=443): Max retries exceeded with url: /new/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
pontofrio.com.br,"HTTPConnectionPool(host='www.pontofrio.com.br', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.pontofrio.com.br', port=80): Read timed out. (read timeout=2)"
popcornfor2.com,"HTTPSConnectionPool(host='popcornfor2.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='popcornfor2.com', port=80): Read timed out. (read timeout=2)"
popvortex.com,"HTTPSConnectionPool(host='popvortex.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'popvortex.com' doesn't match either of '*.gridserver.com', 'gridserver.com'"")))","HTTPConnectionPool(host='www.popvortex.com', port=80): Read timed out. (read timeout=2)"
poradnikogrodniczy.pl,"HTTPSConnectionPool(host='poradnikogrodniczy.pl', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa3f9d79700>, 'Connection to poradnikogrodniczy.pl timed out. (connect timeout=2)'))","HTTPConnectionPool(host='poradnikogrodniczy.pl', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa3f97ebaf0>, 'Connection to poradnikogrodniczy.pl timed out. (connect timeout=2)'))"
porezna-uprava.hr,"HTTPSConnectionPool(host='porezna-uprava.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3f9be8370>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='porezna-uprava.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3f2e9e7c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
port2port.co.il,"HTTPSConnectionPool(host='port2port.co.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.port2port.co.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
portal.gov.bd,"HTTPSConnectionPool(host='portal.gov.bd', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='portal.gov.bd', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
portaldasfinancas.gov.pt,"HTTPSConnectionPool(host='portaldasfinancas.gov.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3f8f9a0d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='portaldasfinancas.gov.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3f9314940>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
portaldeangola.com,"HTTPSConnectionPool(host='www.portaldeangola.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3f9a469d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPSConnectionPool(host='www.portaldeangola.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3fb88ab50>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
portorecanaticalcio.it,"HTTPSConnectionPool(host='www.portorecanaticalcio.it', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.portorecanaticalcio.it', port=443): Read timed out. (read timeout=2)"
portugal.gov.pt,"HTTPSConnectionPool(host='www.portugal.gov.pt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.portugal.gov.pt', port=443): Read timed out. (read timeout=2)"
positioningmag.com,"HTTPSConnectionPool(host='positioningmag.com', port=443): Read timed out.","HTTPSConnectionPool(host='positioningmag.com', port=443): Read timed out. (read timeout=2)"
posted.co.rs,"HTTPSConnectionPool(host='posted.co.rs', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3f40577f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='posted.co.rs', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa3fad980a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
pozitivke.net,"HTTPSConnectionPool(host='pozitivke.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'pozitivke.net' doesn't match either of '*.mojhost.net', 'mojhost.net'"")))","HTTPConnectionPool(host='pozitivke.net', port=80): Read timed out. (read timeout=2)"
